python/pyspark/sql/tests/test_udtf.py

```diff
@@ -69,6 +69,9 @@
 )
 
 
+@unittest.skipIf(
+    not have_pandas or not have_pyarrow, pandas_requirement_message or pyarrow_requirement_message
+)
 class BaseUDTFTestsMixin:
```

Comment:
BaseUDTFTestsMixin shouldn't require pandas dependency. Is this because of the assertDataFrameEqual usage?

Reply:
Could you check https://github.com/apache/spark/pull/44778#discussion_r1464191332? Seems like assertDataFrameEqual does not depend on pandas

Reply:
Yeah.

Reply:
> nvm, we just merged the PR to fix it.

Okay, let me rebase it.